modules/apps/29-fee/module.go

```diff
@@ -7,14 +7,17 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
 
 	"cosmossdk.io/client/v2/autocli"
 	"cosmossdk.io/core/appmodule"
+	appmodulev2 "cosmossdk.io/core/appmodule/v2"
```

Comment:
nit: let's use the `appmodule` (not /v2) path because these modules are all using the non-v2 style of module building

Reply:
its confusing to have both v2 and non v2 imports here

Reply:
[appmodulev2 -> appmodule](https://github.com/cosmos/ibc-go/pull/7866/commits/2132fe5e75799b83da370fc4f34dd78b5b75aea2)